Makefile

```diff
@@ -29,58 +29,86 @@ help:
 	@echo "Missing '.env' file. Creating one using 'env.example' as a template"
 	@cp env.example .env
 	@echo "Please open the '.env' file and adjust to your local setup!"
+	@exit 1
```

Comment:
It's good anyway to give the dev pause, and force them to edit the file before doing anything else.